modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/PartitionCountersNeighborcastFuture.java

```diff
@@ -111,15 +112,22 @@ public void init() {
                 cctx.io().send(n, new PartitionCountersNeighborcastRequest(cntrs, futId, tx.topologyVersion()), SYSTEM_POOL);
             }
             catch (IgniteCheckedException e) {
-                if (!(e instanceof ClusterTopologyCheckedException))
-                    log.warning("Failed to send partition counters to remote node [node=" + peer + ']', e);
-                else
+                if (e instanceof ClusterTopologyCheckedException)
                     logNodeLeft(peer);
+                else
+                    log.warning("Failed to send partition counters to remote node [node=" + peer + ", txId=" + tx.nearXidVersion() + ']', e);
 
                 miniFut.onDone();
             }
         }
 
+        if (log.isInfoEnabled()) {
+            log.info("Starting delivery partition countres to remote nodes [txId=" + tx.nearXidVersion() +
+                ", futId=" + futId +
+                ", topVer=" + tx.topologyVersion() +
+                ", msgs=" + peerMsgs);
```

Comment:
Should we add here `[` `]` for list of messages?

Reply:
list is wrapped by brackets by default, but actually I forgot to add the last closing bracket, thanks for making me double check.